HTTPService/pkg/user/transport.go

```diff
@@ -0,0 +1,79 @@
+package user
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	erro "github.com/coding-kiko/GoKit-Project-Bootcamp/GRPCServiceA/pkg/errors"
+	ent "github.com/coding-kiko/GoKit-Project-Bootcamp/HTTPService/pkg/entities"
+	kitHttp "github.com/go-kit/kit/transport/http"
+	"github.com/go-kit/log"
+	"github.com/gorilla/mux"
+)
+
+func NewMuxApi(endpoints Endpoints, logger log.Logger) http.Handler {
+	// options to support error control
+	options := []kitHttp.ServerOption{
+		kitHttp.ServerErrorLogger(logger),
+		kitHttp.ServerErrorEncoder(encodeErrorResponse), // encode error to the http.ResponseWriter whenever one is found processing a request
+	}
+	r := mux.NewRouter()
+	// GetUser method handler
+	r.Methods("GET").Path("/api/users/{Id}").Handler(
```

Comment:
```suggestion
	r.Methods("GET").Path(GetUser).Handler(
```
place the path stirng in const, and  user thebest practices about rest endpoint naming 
use /User/{Id}  instead /api/users/{id}  (userS, in plural is uses  when you hace a collection)